doc/conf.py

```diff
@@ -321,7 +321,7 @@ def __getattr__(cls, name):
     'matplotlib', 'matplotlib.pylab', 'tifffile', 'EdfFile', 'netCDF4',
```

Comment:
pass will force people to change their scripts; the current arrangement will send a warning but still work. I don't think we have  a big crowd using these importers so we can go with the pass, the only concern I have is the astra/tomopy paper that is just coming out with example using the older call; for this is probably worth it to keep the transition in 2 steps. Perhaps @dmpelt has some comments.